oe_theme.theme

```diff
@@ -1504,6 +1505,25 @@ function oe_theme_theme_suggestions_page_alter(array &$suggestions, array $varia
   }
 }
 
+/**
+ * Implements hook_ui_patterns_suggestions_alter().
+ */
+function oe_theme_ui_patterns_suggestions_alter(array &$suggestions, array $variables, PatternContext $context) {
```

Comment:
See above, if we use the field group template suggestions we don't need to implement this hook here.

Reply:
Fixed (made a commit to keep previous version)